File: Zipping and Unzipping files in .NET application/[C#]-Zipping and Unzipping files in .NET application/C#/ArchiveComment.cs
using System.Windows.Forms;

namespace ArchiveManager
{
    public partial class ArchiveComment : Form
    {
        public ArchiveComment()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Gets or sets the archive's comment.
        /// </summary>
        public string Comment
        {
            get { return txtComment.Text; }
            set { txtComment.Text = value; }
        }
    }
}

File: Zipping and Unzipping files in .NET application/[VB]-Zipping and Unzipping files in .NET application/VB/ArchiveComment.vb
Namespace ArchiveManager
	Partial Public Class ArchiveComment
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		''' <summary>
		''' Gets or sets the archive's comment.
		''' </summary>
		Public Property Comment() As String
			Get
				Return txtComment.Text
			End Get
			Set(ByVal value As String)
				txtComment.Text = value
			End Set
		End Property
	End Class
End Namespace